neo_batterylevelshutdown/neo_batterylevelshutdown.py

```diff
@@ -37,11 +37,18 @@ def setup_gpio_pin(pinNum, direction):
     if not os.path.isfile(pinPath):
         try:
             # Export pin for access
-            with open(GPIO_EXPORT_FILE, "w") as f:
-                f.write(pinNum)
+            # with open(GPIO_EXPORT_FILE, "w") as f:
```

Comment:
Please add a comment to review whether we can use open + write instead of os.system once we have test devices.